package_enterprise_test.go

```diff
@@ -0,0 +1,69 @@
+// +build all enterprise
+
+package main
+
+import (
+	"os"
+	"testing"
+
+	. "github.com/playscale/goconvey/convey"
+)
+
+func TestPackageEnterprise(t *testing.T) {
+	Account := os.Getenv("CONJUR_ACCOUNT")
+	Login := os.Getenv("CONJUR_AUTHN_LOGIN")
+
+	ApplianceURL_V4 := os.Getenv("CONJUR_V4_APPLIANCE_URL")
+	SSLCert_V4 := os.Getenv("CONJUR_V4_SSL_CERTIFICATE")
+	APIKey_V4 := os.Getenv("CONJUR_V4_AUTHN_API_KEY")
+
+	Path := os.Getenv("PATH")
+
+	Convey("Given a compiled summon-conjur package", t, func() {
```

Comment:
I still think this one could go away, too. Unless, somehow, we're going to try to run tests when there isn't a "compiled summon-conjur package"?

Reply:
Sure I'll remove it - we're explicitly creating the package with `build` before the tests.